Polygon_mesh_processing/doc/Polygon_mesh_processing/NamedParameters.txt

```diff
@@ -152,6 +152,30 @@ the property map containing information about edges of the input polygon mesh be
 a default property map where no edge is constrained is provided.
 \cgalNPEnd
 
+\cgalNPBegin{ maximum_number_of_patches } \anchor PMP_maximum_number_of_patches
+contains the surface patch beeing treated +1.\n
+\b Type : `std::size_t`* \n
```

Comment:
Actually it appears it is pretty useless and we will just remove it and return the value instead.